### ISSUE TITLE
TransferWindow Hits

### ISSUE THREAD

**Issue by jens-maus** [jens-maus/yam#148]
**Originally by _opionline@yahoo.de_ on 2010-03-25 11:33:20 +0100**

___
While getting mails with the latest nightly build
YAM 2.7-dev [OS3/m68k] (25.03.2010)
i get 2 MuForce Hits. I attach the hits as TransferWindow-Hit.txt
together with YAMs debugoutput. So you can see wehn the hit occur.

The output from GccFindHit for the first 3 lines are:

11.DFU-YAM:YAM> fh YAM.debug 0015F328
extrasrc/GetHead.c: line 36, offset 0x15f31e

11.DFU-YAM:YAM> fh YAM.debug 0003EE02
mui/TransferWindow.c: line 713, offset 0x3edf0

11.DFU-YAM:YAM> fh YAM.debug 000AE4B6
YAM_MA.c: line 2490, offset 0xae4b6

The 5 Mails waiting on the server are not downloaded.
Afterwards it's not possible to try again to get mails because
the Buttons and menues for that are disabled so i must quit YAM.

Additional: After quit YAM i get a log entry like:
** Cleaning up memory tracking *************************************
Debug.c:831:there are still 1 unfreed memory trackings
mui/TransferWindow.c:193:unfreed memory tracking: 0x0b6ffbf8, size/type 1194, func (calloc)
** Cleaned up debugging ********************************************

**Comment by jens-maus** [comment]
**Originally by _opionline@yahoo.de_ on 2010-03-25 11:33:52 +0100**

___
**Attachment added:** [`TransferWindow-Hit.txt`](https://yam.ch/attachments/148/TransferWindow-Hit.txt) (18.8 KiB)

**Comment by jens-maus** [comment]
**Originally on 2010-03-25 11:41:27 +0100**

___
No need to attach these hits or mention anything transferwindow/preselection window related these days. We know the nightly build is broken and that's why we have that large WARNING: in the ChangeLog :)

This is work in progress and won't get fixed soon. All the preselection window related stuff will change dramatically until we have 2.7 ready and now we are in progress of redesigning everything.

So bottom line: We know about these things and until that preselection/transfer window stuff is finished the nightly builds are known to behave unstable.
